front/src/styles/scss/applications/operationalStudies/_scenarioV2.scss

```diff
@@ -49,7 +49,7 @@
 
         & ~ .close-label {
           text-transform: capitalize;
-          opacity: 1;
```

Comment:
You should keep both the opacity props and the display props, otherwise you have this weird effect (the scenario's name seems to blink)
[Capture vidéo du 2025-05-22 12-04-35.webm](https://github.com/user-attachments/assets/f028fccb-aa32-418c-b9ce-219e8f570307)